src/validateInvocation.test.ts

```diff
@@ -8,15 +8,15 @@ import {
 } from '@jupiterone/integration-sdk-testing';
```

Comment:
Can you rename this file to `getStepStartStates.test.ts`? Also, I would love to see some tests put in place. I could definitely see new roles and/or steps being added in the future and it will help to make sure this functionality is solid when that point comes.